packages/loaders/src/styled/StyledLoadingPlaceholder.ts

```diff
@@ -14,10 +14,10 @@ export const StyledLoadingPlaceholder = styled.div.attrs({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION,
   role: 'progressbar'
-})<{ fontSize: string | number }>`
-  display: inline;
-  width: 1em;
-  height: 0.9em;
+})<{ fontSize: string | number; width?: string; height?: string }>`
+  display: block;
```

Comment:
Are we sure about this? The default display for `<svg/>` (the element that replaces this) is `inline`. The placeholder should mimic that element as closely as possible.

We know for sure these loaders are not meant to take up the full block by default. I would think `inline-block` is the most we could do here.

Reply:
I also noticed `font-size` is bleeding through as a prop to the SVG. That shouldn't be happening, although not exactly the focus of this PR.

Reply:
<img width="1920" alt="Screen Shot 2021-11-02 at 12 45 02 PM" src="https://user-images.githubusercontent.com/1975639/139941836-39951e6d-f8cd-4436-ad3d-65aca85940b8.png">

DevTools says the SVG element is display block. Am I missing something?

FWIW Inline block also achieves the same results in placeholder we are looking for.

Reply:
Hmm, the `display` default for SVGs here – https://zendeskgarden.github.io/svg-icons/?path=/story/12-px--icons – is `inline` 🤔. Also, stripped away https://codepen.io/chriscoyier/pen/gpcDi is `inline`. I can't quite make sense of why the component storybook shows block. But we know that altering all `svg` to `display: block` in the `svg-icons` example would be rather undesirable. I think the same applies here.

If `inline-block` solves for you, then great. Just be sure to [mind the gap](https://dev.to/christiankaindl/the-strange-img-gap-in-html) (not exactly sure if this applies here).